lte/gateway/c/core/oai/tasks/mme_app/mme_app_main.c

```diff
@@ -78,6 +78,9 @@ static int handle_message(zloop_t* loop, zsock_t* reader, void* arg) {
   mme_app_last_msg_latency =
       ITTI_MSG_LATENCY(received_message_p);  // microseconds
   pre_mme_task_msg_latency = ITTI_MSG_LASTHOP_LATENCY(received_message_p);
+  if (pre_mme_task_msg_latency < 0) {
```

Comment:
@ardzoht I notice majority of times `pre_mme_task_msg_latency` is populated with large negative value `-4702111234474983746'. It was always the same negative value.   I think this is populated when there is not a value on pre_mme_task_msg_latency which is obtained from `[ittiMsgHeader.last_hop_latency](https://github.com/magma/magma/blob/master/lte/gateway/c/core/oai/lib/itti/intertask_interface.h#L64)` 

For this cases, I am setting the latency to zero for negative values